app/src/androidTest/java/org/simple/clinic/MigrationAndroidTest.kt

```diff
@@ -391,5 +413,99 @@ class MigrationAndroidTest {
       assertThat(it.count).isEqualTo(2)
     }
   }
-}
 
+  @Test
+  fun migration_17_to_18() {
+    val db_v17 = helper.createDatabase(version = 17)
+
+    val nonExistentFacilityUuid = "non-existent-facility-uuid"
+    val existentFacilityUuid = "464bcda8-b26a-484d-bb70-49b3675f4a38"
+
+    val measurementUuid1 = "ee367a66-f47e-42d8-965b-7a2b5c54f4bd"
+    val prescribedDrugUuid1 = "814132f2-f440-40cb-991c-43861c295815"
+
+    val measurementUuid2 = "fe367a66-f47e-42d8-965b-7a2b5c54f4bd"
+    val prescribedDrugUuid2 = "124132f2-f440-40cb-991c-43861c295815"
+
+    val userUuid = "a1d33096-cea6-4beb-8441-82cab2befe2d"
+    val patientUuid = "464bcda8-b26a-484d-bb70-59b3675f4a38"
+
+    fun insertFacilityStatement(uuid: String, name: String) = """
```

Comment:
For the first time in my life, I see why local functions can be useful sometimes.